src/Workleap.DomainEventPropagation.Abstractions/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,6 @@
-#nullable enable
\ No newline at end of file
+#nullable enable
```

Comment:
Ça va dans Shipped.txt 🙂

Reply:
All `PublicAPI.Shipped.txt` and `PublicAPI.Unshipped.txt` files will be updated once the implementation is stable; keeping this comment open for now.

Reply:
All `PublicAPI.Shipped.txt` and `PublicAPI.Unshipped.txt` have been addressed in #117.